Software/GuitarPedal/Effect-Modules/nam_module.cpp

```diff
@@ -0,0 +1,229 @@
+#include "nam_module.h"
+#include "../Util/audio_utilities.h"
+#include "Nam/model_data_nam.h"
+#include <q/fx/biquad.hpp>
+//#include <math_approx/math_approx.hpp>  // I dont think this is needed, only used in SIMD NAMMathsProvider
+#include "Nam/wavenet/wavenet_model.hpp"
+
+using namespace bkshepherd;
+
+constexpr uint8_t NUM_FILTERS_NAM = 3;
+
+const float minGain = -10.f;
+const float maxGain = 10.f;
+
+const float centerFrequencyNam[NUM_FILTERS_NAM] = {180.f, 1200.f, 5000.f}; // Experiment with these freqs and q values
```

Comment:
[nit] It doesn't need to be changed here if you don't agree, but my ears think I would drop the bass center frequency a tiny bit to 130 or so. The bass adjustment sounds a bit to mid IMO